packages/wallet/backend/src/rapyd/validation.ts

```diff
@@ -36,3 +36,18 @@ export const walletSchema = z.object({
     phone: z.string().optional()
   })
 })
+
+export async function validateRapydResponse<
```

Comment:
I preffered to break the DRY principle here and just copy most of this global method that I previously built for validation of our frontend client requests.